blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractPipelineImpl.java

```diff
@@ -206,6 +212,20 @@ public Resource resolve(Item context, Reachable parent, Item target) {
     }
 
     @Override
+    public int getNumberOfRunningPipelines(){
+        RunList runningJobs =  job.getBuilds().filter(isRunning);
+        return Iterators.size(runningJobs.iterator());
+    }
+
+    @Override
+    public int getNumberOfQueuedPipelines() {
+        if(job instanceof BuildableItem) {
```

Comment:
@cliffmeyers 

>I noticed that if I POST'd repeatedly to the same freestyle job to create a new run, each time the same id and expectedBuilderNumber was returned - like you could only queue the freestyle job once. Is that expected behavior?

to be clear this behavior is different issue not specific to this PR. However @imeredith can answer this question as he has written this part of the code.